api/offer_test.go

```diff
@@ -385,19 +371,43 @@ var _ = Describe("Offer Handler", func() {
 			Expect(obj["error"]).To(Equal("ValidationFailedError"))
 			Expect(obj["description"]).To(Equal("json: cannot unmarshal string into Go value of type models.OfferToUpdate"))
 		})
+
+		It("should not claim if invalid id is passed on body", func() {
```

Comment:
But in this case, if the middleware is not used and I send a correct ID in the body what would happen? Ex
```
POST /offers/valid-id/claim

{
  "id": "another-valid-id",
  "playerId": "player-1",
  "gameId":  "game-1"
}
```

It would claim the offer with id `another-valid-id` correct? I am not sure this would be a good behaviour.